### ISSUE TITLE
output = model(image) 

### ISSUE THREAD

**Issue by G-Apple1** [microsoft/Swin-Transformer#33]
output = model(image)
#output.shape=torch.Size([1, 1000])
output.topk(5,dim=1)#add the code behind 

I use the weight file is swin_tiny_patch4_window7_224.pth

![image](https://user-images.githubusercontent.com/62458945/115200254-9674d380-a126-11eb-9fde-8fe52aef3641.png)

I use a bird image(above) as the model input .
The result of output.topk  is  
**torch.return_types.topk(
values=tensor([[1.7864, 1.6988, 1.6899, 1.6638, 1.5892]]),
indices=tensor([[627, 468, 450,  41, 160]]))**

I want to know the Imagenet output tensor index to label mapping.

**Comment by G-Apple1** [comment]
My fault. I did not load the pre-trained model.
